internal/oidc/const.go

```diff
@@ -40,6 +40,11 @@ const (
 	ClaimClientIdentifier                    = "client_id"
 )
 
+const (
+	// ClientAssertionJWTBearerType is the JWT bearer assertion.
```

Comment:
These were exported due to blackboxing the OIDC tests (to prevent import loops) I think, we can realistically remove them but it's useful to be able to check them like this I think? What do you think?